content/blog/pulumi-bloggers-oct-2019/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Four Infrastructure as Code Blog Posts You Should Read"
+authors: ["sophia-parafina"]
+tags: ["Pulumi","bloggers",]
+meta_image: "blog.png"
+meta_desc: "Continuous delivery of Pulumi apps with Octopus Deploy "
+date: "2019-10-24"
+---
+
+We are always excited when people join the Infrastructure as Code community and write about their experiences. Here are four new posts that we think are great!
+
+## CircleCI on Google Kubernetes Engine (GKE)
+
+[Angel Rivera](https://twitter.com/punkdata) shows how to set up CircleCI on GKE with a [step-by-step tutorial](https://circleci.com/blog/automate-releases-from-pipelines-using-infrastructure-as-code/). Angel demonstrates how to integrate Infrastructure as Code solutions, such as Pulumi, into CI/CD pipelines. The tutorial builds a container with a Pulumi project with a sample app written in Python that deploys Kubernetes. The tutorial shows how you can incorporate your infrastructure into your CI/CD pipeline. The code for the tutorial is available in [GitHub](https://github.com/datapunkz/orb-pulumi-gcp).
```

Comment:
`integrate Infrastructure as Code solutions, such as Pulumi, into CI/CD pipelines`
and 
`The tutorial shows how you can incorporate your infrastructure into your CI/CD pipeline.`
are a bit repetitive!

Reply:
should it be "on GitHub" instead of "in"? (Could totally be wrong here)